be/src/vec/aggregate_functions/aggregate_function_min_max.cpp

```diff
@@ -56,6 +56,11 @@ AggregateFunctionPtr create_aggregate_function_single_value(const String& name,
                 AggregateFunctionsSingleValue<Data<SingleValueDataString>>>(argument_types,
                                                                             result_is_nullable);
     }
+    if (which.idx == TypeIndex::JSONB) {
```

Comment:
min/max is also register by this create_aggregate_function_single_value function,
maybe have error for min/max?